process/project_proposals.md

```diff
@@ -11,32 +11,20 @@ This governance policy sets forth the proposal process for projects to be accept
-
-![Sandbox process](sandbox-process.png)
-
- *Project Proposal*
-   * Project proposed through a [form](https://docs.google.com/forms/d/1bJhG1MuM981uQXcnBMv4Mj9yfV5_q5Kwk3qhBCLa_5A/edit)
-
- *TOC Review*
-   * TOC reviews proposal in spreadsheet and presentation, project passes for inclusion with a simple majority vote. 
-   * TOC may engage with project to ask further questions
-   * Projects may re-apply after six months
-   * Annual reviews are performed in order to check in with sandbox projects, ascertain their progress, and address any outstanding questions
-     * A TOC representative volunteers to lead the review once the project files a PR.
-     * The assigned member reviews the content of the PR and analyzes the project for community health indicators, their findings are placed within a thread in the private TOC channel for discussion
-       * findings should highlight important facts about the project that could influence the TOCs decision around the future of the project, its status in sandbox, its path to incubation, etc.
-       * the thread should always include whether the Project's view of themselves and the ask of the CNCF is reasonable to assist the project moving forward.
-     * Projects are invited to the public TOC meeting to engage in TOC led discussion around the project. Projects are not obligated to attend.
-     * The assigned TOC member provides a summary of the project and leverages the thread's content as the basis of discussion.
-       * Discussion typically focuses on what is going well with the project and areas to improve
-     * Project are invited to use this time to voice any concerns and requests for help they may have that are not captured in the PR (or highlight asks within the PR).
-     * At the conclusion of the public meeting, the TOC votes to approve all annual reviews covered. Should a concern registered on a project, the votes will be held separately.
-     * After the meeting wraps up, the assigned TOC member may summarize the discussion on the PR in the form of a comment to document information for the project and community.
+**Note: The TOC has changed the sandbox application process to a more transparent and streamlined workflow within the :package: [Sandbox Applications repository](https://github.com/cncf/sandbox) :package:.**
+
+All exceptions and "declined" or "postponed" outcomes are handled by the TOC. Projects may be encouraged to re-apply after addressing areas called out in the application comments on the corresponding issue. Please refer to the instructions in the [Sandbox repo README](https://github.com/cncf/sandbox) for more details on re-application.
+
+![Sandbox process](sandbox-application-process-2022.png)
+
+#### Applying for Sandbox
+   * Projects apply for sandbox through the Sandbox Repo's *[Issue Form](https://github.com/cncf/sandbox/issues/new)*. More information on this process is found on the main [Sandbox repo page](https://github.com/cncf/sandbox).
+
```

Comment:
Ditto. I do not see "More information". Aer you going to add it later?

Reply:
It's because the PR on the sandbox repo needs merged with that content.  Go to the sandbox project and look at PR#1's readme update, that should have everything referenced in this PR.